### PR TITLE
Dio 5 support

### PR DESCRIPTION
**Pull request by Carapacik**

### CODE REVIEW

**Comment by Carapacik** [comment]
#5

**Comment by Carapacik** [comment]
Also check my branch with reformat code
https://github.com/Carapacik/flutter_pretty_dio_logger/tree/feature/reformat-code

File: lib/flutter_pretty_dio_logger.dart [hunk 1]
```diff
@@ -107,8 +108,8 @@ class PrettyDioLogger extends Interceptor {
       requestHeaders['contentType'] = options.contentType?.toString();
       requestHeaders['responseType'] = options.responseType.toString();
       requestHeaders['followRedirects'] = options.followRedirects;
-      requestHeaders['connectTimeout'] = options.connectTimeout;
-      requestHeaders['receiveTimeout'] = options.receiveTimeout;
+      requestHeaders['connectTimeout'] = options.connectTimeout?.toString();
+      requestHeaders['receiveTimeout'] = options.receiveTimeout?.toString();
```
Comment (CuongNV12) [inline lib/flutter_pretty_dio_logger.dart:112]: This is not the correct solution

File: pubspec.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 name: flutter_pretty_dio_logger
 description: Show the correct json format, cUrl, easy to copy and use for many purposes.
-version: 2.0.3
+version: 2.1.0
```
Comment (CuongNV12) [inline pubspec.yaml:3]: This is not compliance Package versioning

**Comment by CuongNV12** [comment]
This has been fixed, thank for your effort!

**Pull request closed without merging**